src/app/features/search/multiple-search-results/multiple-search-results.component.spec.ts

```diff
@@ -69,6 +83,24 @@ describe('MultipleSearchResultsComponent', () => {
     }));
   });
 
+  describe('when searchResults is null', () => {
+    beforeEach(() => {
+      store = TestBed.inject(MockStore);
+      techRecordHttpService = TestBed.inject(TechnicalRecordHttpService);
+      store.overrideSelector(selectQueryParams, { vin: '123456' });
+
+      fixture = TestBed.createComponent(MultipleSearchResultsComponent);
+      component = fixture.componentInstance;
+
+      fixture.detectChanges();
+    });
```

Comment:
do you need to re-create the component?

Reply:
this test may not need a new describe either. Could  the 'default tests' describe have the two 'it' conditions - when there are search results and when there are none

Reply:
> do you need to re-create the component?, not convinced you need the beforeEach here

Yes mate. I will get rid of those lines